frontend/app/utils/route-utils.ts

```diff
@@ -100,3 +104,17 @@ export function getRouteByPath(pathname: string, routes: I18nRoute[]): I18nPageR
 function normalizePath(pathname: string): string {
   return pathname.replace(/\/+$/, '');
 }
+
+/**
+ * Create a redirect response using a given I18nRouteFile file.
+ *
+ * @param i18nRouteFile - The file name of the route to search for.
+ * @param request - The HTTP request object containing the URL from which to extract the locale.
+ * @returns redirect response.
+ */
+export function appRedirect(i18nRouteFile: I18nRouteFile, request: Request) {
+  const route = getRouteByFile(i18nRouteFile, i18nRoutes);
+  const lang = getLanguage(request) ?? 'en';
+  const path = route.paths[lang];
+  return redirect(path);
+}
```

Comment:
See #122. I renamed this to `i18nRedirect()` and moved it into `.server/` (as per @sebastien-comeau's suggestion).